src/System.Net.Security/tests/FunctionalTests/SslStreamAlpnTests.cs

```diff
@@ -190,8 +191,7 @@ public async Task SslStream_StreamToStream_Alpn_NonMatchingProtocols_Fail()
         }
 
         [OuterLoop("Uses external server")]
-        [PlatformSpecific(TestPlatforms.OSX)]
-        [Theory]
+        [ConditionalTheory(nameof(ClientSupportsAlpn))]
         [MemberData(nameof(Http2Servers))]
         public async Task SslStream_OSX_Alpn_Success(Uri server)
```

Comment:
Now that the test isn't platform specific, the name should probably be changed.  s/OSX/Client/, perhaps?